integrations/tensorflow/e2e/keras/applications/CMakeLists.txt

```diff
@@ -0,0 +1,138 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# integrations/tensorflow/e2e/keras/applications/BUILD                         #
+#                                                                              #
+# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
+# CMake-only content.                                                          #
+#                                                                              #
+# To disable autogeneration for this file entirely, delete this header.        #
+################################################################################
+
+iree_add_all_subdirs()
+
+file(GLOB _GLOB_X_TEST_PY LIST_DIRECTORIES false RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} CONFIGURE_DEPENDS *_test.py)
+iree_e2e_cartesian_product_test_suite(
+  NAME
+    large_cifar10_tests
+  MATRIX_KEYS
+    "src"
+    "reference_backend"
+    "data"
+    "model"
+    "target_backends"
+  MATRIX_VALUES
+    "applications_test.py"
+    "tf"
+    "cifar10"
+    "MobileNet;MobileNetV2;ResNet50;VGG16;VGG19"
+    "tf;tflite;iree_vmla;iree_llvmaot;iree_vulkan"
+  LABELS
+    "manual"
+)
+
+iree_e2e_cartesian_product_test_suite(
+  NAME
+    enormous_cifar10_tests
+  MATRIX_KEYS
+    "src"
+    "reference_backend"
+    "data"
+    "model"
+    "target_backends"
+  MATRIX_VALUES
+    "applications_test.py"
+    "tf"
+    "cifar10"
+    "DenseNet121;DenseNet169;DenseNet201;NASNetLarge;NASNetMobile;ResNet50V2;ResNet101;ResNet101V2;ResNet152;ResNet152V2"
+    "tf;tflite;iree_vmla;iree_llvmaot;iree_vulkan"
+  FAILING_CONFIGURATIONS
+    ",,,NASNetLarge,iree_vmla"
+    ",,,NASNetMobile,iree_vmla"
+    ",,,NASNetLarge,iree_llvmaot"
+    ",,,NASNetLarge,iree_vulkan"
+    ",,,NASNetMobile,iree_llvmaot"
+    ",,,NASNetMobile,iree_vulkan"
+    ",,,ResNet50V2,iree_llvmaot"
+    ",,,ResNet50V2,iree_vulkan"
+    ",,,ResNet101V2,iree_llvmaot"
+    ",,,ResNet101V2,iree_vulkan"
+    ",,,ResNet152V2,iree_llvmaot"
+    ",,,ResNet152V2,iree_vulkan"
+  LABELS
+    "guitar"
+    "manual"
+    "nokokoro"
+    "notap"
```

Comment:
Any plans to run these sorts of large tests (including the non-hermetic ones) periodically on open source CI?

Reply:
especially as IREE gets faster this would probably be a good idea. since these tests (and especially the model tests) end up testing a lot of the same things, one of them not running due to a network or other non-hermetic failure tends to be okay. so we could get around the (rare) possibility of failure at expected flaky points by providing no status (or no status in cmake, and failure in bazel since bazel might re-run the test?).

Reply:
No specific plans, but it seems like a good idea.

Reply:
Yeah my main complaint is flakiness that causes spammy failure notifications. Don't want to add yet another source of flakiness to the things the build cop has to monitor.